src/components/Location/Location.js

```diff
@@ -94,18 +107,25 @@ export const Location = ({ savedTides, setSavedTides, setError, setServerDown })
           {convertDecimalToFeetAndInches(tide.height)}
         </td>
         <td className="center">
-          <button
+          <img
+            alt="Star, click to favorite"
+            src={star}
             onClick={(e) => handleClick(e, tide, location)}
             className="tide-save-l"
-          ></button>
+          ></img>
         </td>
       </tr>
     );
   });
 
-  return (
-    <main className={`${location.station} main-page`}>
-      <h1>{location.name}</h1>
+  const renderMain = () => {
+    return (
+      <div>
```

Comment:
Is this div doing anything?

Reply:
JSX elements need to be wrapped in a single element.